cmd/kubeadm/app/cmd/selfhosting/BUILD

```diff
@@ -29,11 +29,10 @@ go_library(
         "selfhosting.go",
         "selfhosting_volumes.go",
     ],
-    importpath = "k8s.io/kubernetes/cmd/kubeadm/app/phases/selfhosting",
+    importpath = "k8s.io/kubernetes/cmd/kubeadm/app/cmd/selfhosting",
```

Comment:
Package `k8s.io/kubernetes/cmd/kubeadm/app/phases/selfhosting` should not be moved under `/cmd`.
Might be you lack a bit of context here, but you can consider `/cmd` UX (cobra commands) and `/app/phases` as business logic (tl dr; I think that soon or later we should move the latest to a less ambiguous `/pkg`, but this wasn't discussed yet)

Reply:
i pretty much think this first PR should have only done the removal of deprecated F-gates.

Reply:
@fabriziopandini I thought this `/phases/selfhosting` was going to become the backbone of the new command `kubeadm selfhosting pivot` is this not the case?

Reply:
@Klaven yes, `/phases/selfhosting` should be used as a backbone for the new command `kubeadm selfhosting pivot`, but it should remain where it is (it should not be moved under `kubeadm/app/cmd`)

Reply:
Ok, awesome. I will update it like this. sorry about that! Are we leaving it a phase then? that seems odd as we are moving it out of being a phase. do you want me to move it somewhere else or just leave it?